docs/releases/breaking-changes.md

```diff
@@ -21,13 +21,62 @@ The library `exif-reader`, which is used for Exif metadata extraction of images,
 release, tag names have been updated to align with the Exif standard. See
 https://github.com/devongovett/exif-reader/pull/30 for a complete list of updated tags.
 
-This might be a breaking change if
-
-- a custom `FILE_METADATA_ALLOW_LIST` config is in place
-- you rely on the generated Exif tags stored in Directus Files
+This might be a breaking change if a custom `FILE_METADATA_ALLOW_LIST` config is in place, or you rely on the generated
+Exif tags stored in Directus Files to not change name.
 
 The updated Exif tags only apply to images which are uploaded after upgrading to this release.
 
+::: details Migration/Mitigation
+
+This change updates the following file metadata property names:
+
+::: code-group
+
+```md [Before]
+- image
+- thumbnail
+- exif
+- gps
+- interoperability
+```
+
+```md [After]
+- Image
+- Thumbnail
+- Photo
+- GPSInfo
+- Iop
+```
+
+:::
+
```

Comment:
This is internal-only, the actual changes users might be affected from are listed under https://github.com/devongovett/exif-reader/pull/30. 

```suggestion
```